docs/teachertool/catalog-shared.json

```
@@ -11,6 +11,18 @@
             "use": "custom_function_called",
             "template": "A custom function exists and gets called",
             "docPath": "/teachertool"
+        },
+        {
+            "id": "76D86387-E3CF-4BEB-B62C-B811F5997631",
+            "use": "block_comment_used",
+            "template": "At least one comment was left on a block in the code",
+            "docPath": "/teachertool"
+        },
+        {
+            "id": "8F97C9A6-CF16-48B4-A84F-3105C24B20DE",
+            "use": "functions_have_comments",
+            "template": "All function definitions have block comments",
```

I worry the term "block comment" is going to be confusing for teachers, and in this case it's redundant anyway.
```suggestion
            "template": "All function definitions have comments",
```

---

Yeah, the thing I worry about is that if a teacher looks at a project that has workspace comments next to a function definition that pertains to the function and we fail the validation, it would seem broken.

---

I'm not sure we preserve workspace comment locations well enough for that to work regardless. In this case I'd personally opt for the simpler wording and see if we get feedback later on.

I think the term "block comment" might just make teachers confused. To me, it sort of sounds like some comment that's an actual block in the code.